src/dashboard/header/DashboardHeaderPageTitle.tsx

```diff
@@ -19,7 +19,7 @@ export const DashboardHeaderPageTitle = ({ key, title, tabIndex, removePage, set
   const handleMenuEditClick = (event: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLElement>) => {
     event.preventDefault();
     if (editing) {
-      debouncedSetPageTitle(key, titleText);
+      debouncedSetPageTitle(tabIndex, titleText);
```

Comment:
We should fix the issue where editing is also possible for pages you are not on - i suspect it might be related to the duplicate uuid bug

Reply:
Well, this fixes the fact editing another page name was changing the name of the principal page. Do we wanna remove completely the chance of updating titles this way considering the nowadays full refresh of the page?

Reply:
Cool nevermind, this is good!